src/main/java/Deadline.java

```diff
@@ -0,0 +1,52 @@
+public class Deadline extends Task {
+    protected String by;
+    public Deadline (String description, String by) throws EmptyDescriptionException {
```

Comment:
There is a white space between Deadline and (String description, String by), it should be Deadline(String description, String by).